manta/auth.py

```diff
@@ -183,21 +218,27 @@ def load_ssh_key(key_id, skip_priv_key=False):
         pub_key_path=pub_key_path,
         fingerprint=fingerprint,
         priv_key_path=priv_key_path,
-        priv_key=priv_key)
+        priv_key=priv_key,
+        algorithm=algo)
+
+def rsa_sig_from_agent_sign_response(response):
```

Comment:
Consistency nits: "...from_agent_sign_response" here, but from "...*signed*..." response in the next function. 
Also:
- `response` v `d`
- `algo` v `alg`

Reply:
done.